docs/src/components/InteractiveExample/index.tsx

```diff
@@ -93,7 +93,6 @@ export default function InteractiveExample({
   const [tab, setTab] = React.useState<Tab>(Tab.PREVIEW);
   const [jsxCode, setJsxCode] = React.useState(() => compileTSXtoJSX(tsxCode));
   const [width, setWidth] = React.useState<number | null>(null);
-  const [showPreview, setShowPreview] = React.useState(!showCode);
```

Comment:
`showCode` is now marked as unused, are you sure it should be removed in this PR?

Reply:
yes, because apparently showPreview was unused as well